load-test/README.md

```diff
@@ -1,62 +1,17 @@
-# New Relic's Prometheus Load Generator
```

Comment:
This is what I mean in the PR description, git is comparing to different READMEs